rest/views.py

```diff
@@ -48,6 +49,39 @@
 from django.http.response import Http404
 
 
+def get_urls(raw_urls, list_urls, urlbase=''):
+    """
+    Recursively builds a list of all the urls in the current project and
+    the name of their associated view
+    """
+    list_urls = list_urls or []
+    from operator import itemgetter
```

Comment:
just run tests on my local instance of docker and everything looks fine. I moved the line in this way

```
from __future__ import unicode_literals

from operator import itemgetter

from django.contrib.auth.models import User
```